tasks/setup_environment.yml

```diff
@@ -17,7 +17,7 @@
       - "Environment: {{ env }}"
       - "Type: {{ type }}"
       - "Client: {{ client }}"
-      - "Version: {{ vars[project][client].version }}"
+      - "Version: {{ vars[project][client].version | default(vars[project][type].version, true) }}"
```

Comment:
is true really required ? either it would be defined or undefined I think

Reply:
Also we might have to updated the service_identifier logic to include cluster for ssv